### PR TITLE
added ADR for CRA

### PR DESCRIPTION
**Pull request by breanneboland**
Attempted to revive history in an ADR for our choice to use CRA.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "adr-log": "^2.1.1",
```
Comment (tinyels) [inline package.json]: I don't think you need this now. (It's in dev-dependencies in master)

**Comment by breanneboland** [comment]
@akostibas and @tinyels, what do you think?

File: package.json [hunk 2]
```diff
@@ -25,7 +25,6 @@
   },
   "version": "0.1.0",
   "devDependencies": {
-    "adr-log": "^2.1.1",
```
Comment (akostibas) [inline package.json]: I think we want to keep this in

**Comment by akostibas** [comment]
Looks like you removed the index? If you're adding an ADR, I believe you need to submit a modified `index.md`, right?

**Pull request merged**